modules/tensor_mechanics/include/utils/Guarantee.h

```diff
@@ -0,0 +1,21 @@
+/****************************************************************/
```

Comment:
Would it be better to name this something like 'PropertyGuarantee' just to be a little more descriptive about what it does?  I don't want to make everything have huge names, and I'm not that set on renaming it, so I'll leave that up to you.